docs/conf.py

```diff
@@ -44,10 +49,25 @@
 master_doc = "index"
 
 # General information about the project.
+_today = date.today()
 project = "Towncrier"
-copyright = "2017, Amber Brown"
+copyright = "{}, Towncrier contibutors. Ver {}. Built on {}".format(
+    _today.year,
+    towncrier_version.public(),
+    _today.isoformat(),
```

Comment:
I believe this is problematic, because it doesn't allow for reproducible builds. Maybe git ref instead?

Reply:
Thanks for the review.

Is this a big problem for the docs?
In practice, I expect that we generate the docs in Read the Docs once, and forget about it.

This was copied from twisted/twisted.

If this is good enough, I prefer to merge it as it is.

Reply:
If the build is reproducible then it won't matter when it was built because whenever it was built, you'll get the same output. :)  This is a pretty nice thing.

If the version of the docs being built is available then I don't think you should need to know the date.

I don't know if the doc build is reproducible even _without_ this change so I don't know how much it matters now - but I would expect reproducibility to be a goal that is eventually worked towards and achieving it will require undoing this change, so perhaps it's better to just remove the date now.

Reply:
I have removed the build date.

We still have the copyright year.

I have also fixed the typo.

I hope this is ready to merge.

Reply:
The year isn't [really necessary either](https://hynek.me/til/copyright-years/), but that takes things too far for this PR